src/miner.cpp

```diff
@@ -142,7 +142,7 @@ CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const CScript& s
       CBlockHeader h;
       nBlockSize += h.GetSerializeSize(SER_NETWORK, PROTOCOL_VERSION);
     }
-    assert(nBlockSize == 80);  // BU always 80 bytes
+    assert(nBlockSize == 80);  // BU block header is always 80 bytes
```

Comment:
good question.  I did hesitate in the case of MAX_COINBASE_SCRIPTSIG_SIZE but that parameter is already defined and used just a few LOC above the change.  In this case AFAIK we need to add a new constant which likely means touching another file.  I'm happy to do this in the next PR.